web/api/netdata-swagger.yaml

```diff
@@ -444,15 +444,15 @@ paths:
             multiple using a pipe with a condition each, like this:
             `color<value|color:null` The following operators are
             supported: >, <, >=, <=, =, :null (to check if no value exists). Each color can be specified in same manner as for `label_color` parameter.
-            Currently only integers are suported as values."
+            Currently only integers are supported as values."
           required: false
           allowEmptyValue: true
           schema:
             type: string
             format: any text
         - name: text_color_lbl
           in: query
-          description: Font color for label *(left)* part of the badge. One of predefined colors or as HTML hexadecimal color without preceeding `#` character. Formats allowed `RGB` or `RRGGBB`. If no or wrong value given default color will be used.
+          description: Font color for label *(left)* part of the badge. One of predefined colors or as HTML hexadecimal color without preceding `#` character. Formats allowed `RGB` or `RRGGBB`. If no or wrong value given default color will be used.
```

Comment:
Review dog may hate this, but it's a pre-existing condition.
yaml files do support a line wrapping thing, and wrapping this file (or similar) could be done in a distinct PR.

Reply:
#10495